src/mercadoPago/initMercadoPago/index.ts

```diff
@@ -38,15 +38,16 @@ function checkOptionObject(oldOption: TOptions, newOption: TOptions): boolean {
  * @param options TOptions
  */
 const initMercadoPago = (publicKey: string, options?: TOptions) => {
+  const injectFrontEndOption = { ...options, frontEndStack: 'react' };
+
   MercadoPagoInstance.publicKey =
     publicKey !== MercadoPagoInstance.publicKey ? publicKey : MercadoPagoInstance.publicKey;
-  if (options) {
-    MercadoPagoInstance.options = !checkOptionObject(MercadoPagoInstance.options, options)
-      ? options
-      : MercadoPagoInstance.options;
-  } else {
-    MercadoPagoInstance.options = {};
-  }
+  MercadoPagoInstance.options = !checkOptionObject(
+    MercadoPagoInstance.options,
+    injectFrontEndOption,
+  )
+    ? injectFrontEndOption
+    : MercadoPagoInstance.options;
```

Comment:
Nesse caso, de usar `MercadoPagoInstance.options`, precisamos injetar também o `frontEndStack`?

Reply:
@icaldana Nesse caso ele está passando por uma comparação e só vai usar o `MercadoPagoInstance.options` se ele for igual ao injectFrontEndOption, o que indica que o frontEndStack já está por lá.